src/plonkish/ir/sc.rs

```diff
@@ -12,6 +12,23 @@ pub struct SuperCircuit<F, MappingArgs> {
     mapping: MappingGenerator<F, MappingArgs>,
 }
 
+impl<F: Debug, MappingArgs: Debug> Debug for SuperCircuit<F, MappingArgs> {
```

Comment:
@rutefig are these implementations different from the derive ones?

Reply:
Only for the `MappingArgs` - removed the one for `SuperCircuit` and added derived one